lib/OpenQA/Shared/Controller/Running.pm

```diff
@@ -113,6 +113,7 @@ sub streamtext ($self, $file_name, $start_hook = undef, $close_hook = undef) {
     # Setup utility function to close the connection if something goes wrong
     my $timer_id;
     my $close_connection = sub ($self) {
+        log_debug "Connection to $worker will be terminated because $logfile changed";
```

Comment:
Can you log the worker id instead? `$worker` as a string will only log the internal memory address